src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.HashMap;
+
+import exception.DukeException;
+import parser.Parser;
+import storage.DataStorage;
+import task.TaskList;
+import ui.Ui;
+
+
 public class Duke {
+    private static TaskList tasks;
+    private static DataStorage storage;
+    private static boolean shouldRun = true;
+
+    /**
+     * Entry point for the Duke program
+     * @param args Command line arguments which are not used. Do not waste your effort.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        startup();
+        Ui.greet();
+
+        while (shouldRun) {
+            try {
```

Comment:
Please refer to the CS2103T coding conventions. Switch and case should be in the same indentation.